upup/models/cloudup/resources/addons/storage-openstack.addons.k8s.io/k8s-1.16.yaml.template

```diff
@@ -280,6 +280,7 @@ spec:
           volumeMounts:
             - name: socket-dir
               mountPath: /var/lib/csi/sockets/pluginproxy/
+        {{ if HasSnapshotController }}
```

Comment:
currently this is default behaviour in kops 1.23.2. The problem is that if now someone just updates cluster with newer kops and he does not have snapshotcontroller true in spec, applying manifest will fail. Or is it still the case @olemarkus ? So we should have way to remove container

Reply:
@olemarkus soo this behaviour is fixed in latest beta? Can you confirm